components/brave_news/browser/feed_v2_builder.cc

```diff
@@ -313,15 +313,14 @@ int PickRouletteWithWeighting(const ArticleInfos& articles,
   double picked_value = base::RandDouble() * total_weight;
   double current_weight = 0;
 
-  uint64_t i;
-  for (i = 0; i < weights.size(); ++i) {
+  for (uint64_t i = 0; i < weights.size(); ++i) {
```

Comment:
size_t would be better as `decletype(size())` would be different per CPU bits.

```suggestion
  for (size_t i = 0; i < weights.size(); ++i) {
```